CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Removed
 
 ### Fixed
+- [BUG] Unable to deserialize AliasDefinition ([#1537](https://github.com/opensearch-project/opensearch-java/pull/1537))
```

Comment:
The bug is rather the inability to deserialize a `wrapper` `Query`.
```suggestion
- Fixed deserializing `wrapper` queries ([#1537](https://github.com/opensearch-project/opensearch-java/pull/1537))
```

If also pulling in the `AliasDefinition` changes, might want to add a separate entry for adding `isHidden` to `AliasDefinition`.